libs/ui/elements/src/lib/record-feature-catalog/feature-catalog-list/feature-catalog-list.component.ts

```diff
@@ -0,0 +1,52 @@
+import { Component, Input, OnInit } from '@angular/core'
+import { CommonModule } from '@angular/common'
+import { TranslateModule } from '@ngx-translate/core'
+import { DatasetFeatureCatalog } from '@geonetwork-ui/common/domain/model/record'
+import {
+  ExpandablePanelComponent,
+  TruncatedTextComponent,
+} from '@geonetwork-ui/ui/layout'
+
+@Component({
+  selector: 'gn-ui-feature-catalog-list',
+  templateUrl: './feature-catalog-list.component.html',
+  standalone: true,
+  imports: [
+    CommonModule,
+    TranslateModule,
+    ExpandablePanelComponent,
+    TruncatedTextComponent,
+  ],
+})
+export class FeatureCatalogListComponent implements OnInit {
+  @Input() filteredFeatureCatalog: DatasetFeatureCatalog
+
+  columns = [
+    {
+      key: 'type',
+      label: 'feature.catalog.attribute.type',
```

Comment:
@Guillaume-d-o Don't forget to use markers when translations are dynamic or not explicitly a translation key; like this case here, otherwise translations will disappear when running the i18n utility (there are missing translation on main currently).